### ISSUE TITLE
a element work break css property

### ISSUE THREAD

**Issue by paskal** [xianmin/hugo-theme-jane#225]
[_content.scss:33](https://github.com/xianmin/hugo-theme-jane/blob/588d5bf/assets/sass/_partial/_post/_content.scss#L33) have following code:
```css
.post .post-content a {
    color: $theme-color;
    word-break: break-all;

    &:hover {
      border-bottom: $content-link-border;
    }
  }
```

With `word-break: break-all;` (break at any character), current behaviour:
![image](https://user-images.githubusercontent.com/712534/62891133-391b6300-bd45-11e9-9790-137b2eff34e9.png)

With `word-break: break-word;` (break by words), current behaviour for everything except links in post:
![image](https://user-images.githubusercontent.com/712534/62891361-ceb6f280-bd45-11e9-96dc-4bfeeb871352.png)

I think latter behaviour is preferable and `word-break: break-all;` should be removed.

**Comment by xianmin** [comment]
I forgot why add `word-break: break-all` here.
But, you are right, it's a problem. We can remove it first.

**Comment by paskal** [comment]
I've checked, it's fixed after #226. Resolving.
